pyproject.toml

```diff
@@ -70,6 +70,8 @@ django-prometheus = "~2.2.0"
 django-redis = "~5.2.0"
 # RQ (Redis Queueing) for background handling of webhooks, jobs, etc.
 django-rq = "~2.5.1"
+# DiscoverSlowestTestsRunner for running CI performance tests and benchmarking.
+django-slowtests = "^1.1.1"
```

Comment:
Does this need to be a core dependency or can we get away with it being a dev dependency?

Reply:
Great point! Same potential issue here as #2536.